highfidelity/hifi
# Migrate Avatar rendering code to library

This migrates the core code for rendering of avatars out of the main application and into an avatars-rendering library.  Application behavior should not be changed. 

## Testing

* Verify that avatars (both your own and others) still render the same way they do in master
* Verify that facial and body animation still works for your own avatar (facial animation should be triggered by audio)
* Verify that facial and body animation still works for other avatars
* Verify camera / menu interaction
  * When a script changes the camera mode (such as when you double click on the entity list and the camera zooms to that entity) does the menu properly update (in the above case it should change to 'independent mode')
